LoopKitUI/Views/Settings Editors/InsulinModelSelectionView.swift

```diff
@@ -0,0 +1,317 @@
+//
+//  InsulinModelSelection.swift
```

Comment:
I'm confused...is this a duplicate file?

Reply:
We love merge conflicts - removed